### ISSUE TITLE
Passing an empty file with `@` syntax now throws an error on 1.7

### ISSUE THREAD

**Issue by jsifuentes** [jpmens/jo#194]
Prior to 1.7, `jo` would place an empty string as the field value rather than throwing an error.

```
touch empty_file
jo "value=@empty_file"
```

On 1.7:
```
jo: Error reading file empty_file
```

Prior to 1.7:
```
{"value":""}
```

Based on the changelog, I think this refactor was the cause of the new behavior: #192 

--

I found this behavior because it broke a script that did something like the following:

```
local stdout="$(mktemp)"
local stderr="$(mktemp)"

brew install somepackage 1>"$stdout" 2>"$stderr"

jo stdout="@$stdout" stderr="@$stderr"
```

When there is nothing in stderr, the resulting file is empty, which now causes `jo` to throw an error.

**Comment by jpmens** [comment]
Thanks for reporting this. I think it’s a bug, as the file is clearly readable:

```console
$ jo data=@/dev/null
jo: Error reading file /dev/null
```

**Comment by gromgit** [comment]
Yeah, sorry about that. I'll relook at that patch in a few hours' time, when I get back home.

**Comment by jpmens** [comment]
Thank you, @gromgit!

@jsifuentes we have cut a [new release 1.8](https://github.com/jpmens/jo/releases/tag/1.8) and would appreciate a short note whether this solves your issue.

**Comment by jsifuentes** [comment]
Confirmed this issue is resolved in jo 1.9. Homebrew still has 1.7, but it looks like the version will be bumped shortly. https://github.com/Homebrew/homebrew-core/pull/114625

Thank you both for your great work!
